lib/taskana-core/src/test/java/acceptance/workbasket/DeleteWorkbasketAccTest.java

```diff
@@ -67,14 +68,18 @@ public void testDeleteWorkbasket()
     }
 
     @WithAccessId(userName = "elena")
-    @Test(expected = NotAuthorizedException.class)
+    @Test
     public void testDeleteWorkbasketNotAuthorized()
         throws WorkbasketNotFoundException, NotAuthorizedException, WorkbasketInUseException, InvalidArgumentException {
-        Workbasket wb = workbasketService.getWorkbasket("TEAMLEAD_2", "DOMAIN_A");
-        workbasketService.deleteWorkbasket(wb.getId());
 
-        workbasketService.getWorkbasket("TEAMLEAD_2", "DOMAIN_A");
-        fail("NotAuthorizedException was expected.");
+        Assertions.assertThrows(NotAuthorizedException.class, () -> {
+            Workbasket wb = workbasketService.getWorkbasket("TEAMLEAD_2", "DOMAIN_A");
+            workbasketService.deleteWorkbasket(wb.getId());
+        });
+
+        Assertions.assertThrows(NotAuthorizedException.class, () ->
```

Comment:
This is either a second test or obsolete.

Reply:
done